### PR TITLE
add OIDC provider to the idp section

### PR DESCRIPTION
**Pull request by edewit**
## Motivation

implements:
     OIDC provider details - General
     OIDC provider details - OIDC settings
     OIDC provider details - Advanced
     github provider details - Advanced

## Checklist:

- [x] Code has been tested locally by PR requester
- [x] User-visible strings are using the react-i18next framework (useTranslation)
- [x] Help has been implemented
- [x] axe report has been run and resulting a11y issues have been resolved
- [x] Unit tests have been created/updated
- [x] Formatting has been performed via prettier/eslint
- [x] Type checking has been performed via 'yarn check-types'

### CODE REVIEW

**Review by ssilvert — CHANGES_REQUESTED** [review]
Help text is wrong for `Post Login Flow` and `Sync Mode`.

Also, the dropdown for `Sync Mode` is partially obstructed.
SyncModeObstructed

File: src/identity-providers/messages.json [hunk 1]
```diff
@@ -25,6 +32,62 @@
...
+    "authorizationUrl": "Authorization URL",
+    "passLoginHint": "Pass login_hint",
+    "passCurrentLocale": "Pass current locale",
+    "tokenUrl": "Token URL",
+    "logoutUrl": "Logout URL",
+    "backchannelLogout": "Backchannel logout",
+    "disableUserInfo": "Disable user info",
+    "userInfoUrl": "User Info URL",
+    "issuer": "Issuer",
+    "scopes": "Scopes",
```
[the remark this answers is not in the document]
Comment (edewit) [reply src/identity-providers/messages.json:55]: It's called scopes in the design? @xianli123

**Comment by edewit** [comment]
@ssilvert don't have the problem that the dropdown is obscured I can still use it:
image
@xianli123 is good like this or do you agree with @ssilvert and should we look at how to change this?

File: src/identity-providers/messages.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (yih-wang) [reply src/identity-providers/messages.json]: Right, the word "default scopes" doesn't fit the scenario. So changed it.

**Comment by yih-wang** [comment]
@edewit I'm good with your screenshot.

File: src/identity-providers/messages.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (ssilvert) [reply src/identity-providers/messages.json]: OK

**Comment by ssilvert** [comment]
> @ssilvert don't have the problem that the dropdown is obscured I can still use it:
> image
> @xianli123 is good like this or do you agree with @ssilvert and should we look at how to change this?

It can still be used, but the bottom of the dropdown and the label are cut off.  Does yours look like mine when the dropdown is closed?

Looks like it just needs some spacing at the bottom of the form.

**Comment by edewit** [comment]
@ssilvert no when I scroll all the way down it looks like this:
image

**Comment by ssilvert** [comment]
> @ssilvert no when I scroll all the way down it looks like this:
> image

@edewit OK.  In that case, let's just fix the conflicts and merge.  We can let this slide for now.

**Review by ssilvert — APPROVED** [review]

**Pull request merged**